presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -279,17 +279,14 @@ Calculating the approximate distinct count can be done much more cheaply than an
     A P4HyperLogLog sketch is similar to :ref:`hyperloglog_type`, but it starts (and remains)
     in the dense representation.
 
-KHyperLogLog
```

Comment:
Please keep the heading as KHyperLogLog is not another type of HyperLogLog, it's a dedicated, separate data structure which fulfills a different purpose than HyperLogLog.